schema/schema.go

```diff
@@ -52,7 +52,7 @@ func (ta *Table) AddColumn(name string, columnType string, extra string) {
 
 	if strings.Contains(columnType, "int") || strings.HasPrefix(columnType, "year") {
 		ta.Columns[index].Type = TYPE_NUMBER
-	} else if columnType == "float" || columnType == "double" {
+	} else if columnType == "float" || columnType == "double" || strings.HasPrefix(columnType, "decimal") {
```

Comment:
can you help me use strings.HasPrefix to check float and double too. the double can be defined like "double(4,2)" too.